src/libsyntax/ext/format.rs

```diff
@@ -452,7 +452,7 @@ impl<'a, 'b> Context<'a, 'b> {
             Some(ecx.lifetime(sp, special_idents::static_lifetime.name)),
             ast::MutImmutable);
         let slice = ecx.expr_vec_slice(sp, pieces);
-        let st = ast::ItemStatic(ty, ast::MutImmutable, slice);
+        let st = ast::ItemConst(ty, slice);
```

Comment:
This is actually intentionally a `static` to reduce codegen times as much as possible (no inlining necessary)